samples/storageclass/isilon.yaml

```diff
@@ -48,6 +48,11 @@ parameters:
   # Optional: true
   #ClusterName: <cluster_name>
 
+  # Sets the filesystem type which will be used to format the new volume
+  # Optional: true
+  # Default value: None
+  #csi.storage.k8s.io/fstype: "nfs"
```

Comment:
Should we also add allowed values in this?

Reply:
as a NFS driver we should not be mentioning other types